opsgenie/resource_opsgenie_team_membership.go

```diff
@@ -0,0 +1,163 @@
+package opsgenie
+
+import (
+	"context"
+	"fmt"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/opsgenie/opsgenie-go-sdk-v2/team"
+	"log"
+	"strings"
+)
+
+func resourceOpsGenieTeamMembership() *schema.Resource { //TODO encode the e-mail addrs (e.g. because of +)? https://github.com/opsgenie/opsgenie-go-sdk-v2/issues/62
```

Comment:
might not become an issue at all. see https://github.com/opsgenie/opsgenie-go-sdk-v2/issues/62